### ISSUE TITLE
javascript error, not found `formLabel`

### ISSUE THREAD

**Issue by hhys** [macbookandrew/cf7-google-analytics#8]
An error will occur on this line.

https://github.com/macbookandrew/cf7-google-analytics/blob/v1.8.4/inc/class-cf7-google-analytics.php#L161

**Comment by macbookandrew** [comment]
Fixed in v1.8.5, just released.

Note: you may want to upgrade your Contact Form 7 plugin—it appears you’re using an old version of CF7.

**Comment by hhys** [comment]
Please confirm...

```diff
diff --git a/inc/class-cf7-google-analytics.php b/inc/class-cf7-google-analytics.php
index 1109f04..d86b725 100644
--- a/inc/class-cf7-google-analytics.php
+++ b/inc/class-cf7-google-analytics.php
@@ -170,7 +170,7 @@ class CF7_Google_Analytics {
                                        gtag( "event", "contact_form_7", {"event_category": "Contact Form 7", "event_action": "Sent", "event_label": "%1$s"} );
                                }
                                if ( typeof dataLayer !== "undefined" ) {
-                                   dataLayer.push({ "event": "Contact Form 7", "event_action": "Sent", "event_label": %1$s });
+                                   dataLayer.push({ "event": "Contact Form 7", "event_action": "Sent", "event_label": "%1$s" });
                                }
                                if ( typeof ga !== "undefined" ) {
                                        ga( "send", "event", "Contact Form", "Sent", "%1$s" );
```

**Comment by macbookandrew** [comment]
Here’s the relevant change: https://github.com/macbookandrew/cf7-google-analytics/commit/9d9c1adc82114fe002f982c4736932b2ef4bc812

**Comment by hhys** [comment]
I think double quotation is necessary.

`%1$s` => `"%1$s"`

**Comment by macbookandrew** [comment]
Fixed in v1.8.7.
